src/Reflow/Parser.hs

```diff
@@ -62,6 +62,12 @@ codeBlock = do
     void eol
     return $ CodeBlock $ s <> c <> e
 
+pgpBlock :: Parser Content
+pgpBlock = fmap PGPBlock $ mappend3 <$> pgpBlockStart <*> pgpBlockContents <*> pgpBlockEnd
```

Comment:
`mappend3` could be used in the `codeBlock` method above, but that felt out of scope.